.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish package to NPM
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - run: yarn install
+      - run: yarn lint:fix
```

Comment:
it is not okay. you should use "fix" option locally before committing. action must **test** linting before building and publishing